Gemfile

```diff
@@ -1,4 +1,3 @@
 source 'https://rubygems.org'
-ruby '2.2.3'
```

Comment:
@wslack Restore this line and change `2.2.3` to `2.3.1` — that should sort it out. 🚀